packages/hop-node/src/watchers/BondTransferRootWatcher.ts

```diff
@@ -167,31 +180,43 @@ class BondTransferRootWatcher extends BaseWatcher {
 
     try {
       const tx = await this.sendBondTransferRoot(
+        transferRootId,
         transferRootHash,
         destinationChainId,
-        totalAmount
+        totalAmount,
+        transferIds,
+        committedAt
       )
+
       const msg = `L1 bondTransferRoot dest ${destinationChainId}, tx ${tx.hash} transferRootHash: ${transferRootHash}`
       logger.info(msg)
       this.notifier.info(msg)
     } catch (err) {
       logger.error('sendBondTransferRoot error:', err.message)
+      if (err instanceof PreTransactionValidationError) {
+        logger.error('pre transaction validation error. turning off writes.')
+        globalConfig.emergencyDryMode = true
```

Comment:
suggestion: use setter function for setting global config values

```suggestion
        enableEmergencyDryMode()
```

in config.ts

```ts
export function enableEmergencyMode() {
 config.emergencyDryMode = true
}
```